### PR TITLE
remove resources no longer created by control_plane module

### PR DESCRIPTION
**Pull request by kevinwang-sfdc**
This change removes some resource creation from the `control_plane` Terraform module which is being moved to the EKS provisioning step.

### CODE REVIEW

File: src/eks_setup_helper.py [hunk 1]
```diff
@@ -313,7 +313,7 @@ def inbound_eks_nlb_setup(deploy_stage, manifest_data):
             print("Created ServiceID: %s, ServiceName: %s, ServiceState: %s" % (svcID, svcName, svcState))
 
             # Create a VPC EndPoint Connection Notification
-            conn_notif_arn_parameter_key = "/{}-{}/{}/inbound-data-plane/vpce-sns-topic".format(
+            conn_notif_arn_parameter_key = "/{}-{}/{}/control-plane/inbound-vpce-sns-topic".format(
```
Comment (agajjala) [inline src/eks_setup_helper.py:316]: @kevinwang-sfdc Can we do a cursory check of all the SSM Values and create them if they are missing

Comment (agajjala) [reply src/eks_setup_helper.py:316]: Screen Shot 2020-06-03 at 1 55 46 PM

☝️  This is the list we need today..

Comment (kevinwang-sfdc) [reply src/eks_setup_helper.py:316]: @agajjala Can you confirm `vpce-svc-id` is needed? I don't see a reference to this in the latest code. Perhaps it was from before we added the change to create the VPCE service via script.

Comment (agajjala) [reply src/eks_setup_helper.py:316]: nope.. it was before.. that can be skipped..
Also forgot about the TGW and VPN-IDs

Comment (kevinwang-sfdc) [reply src/eks_setup_helper.py:316]: Those are covered.

File: Infrastructure/provider/aws/outbound_data_plane/main.tf [hunk 2]
```diff
@@ -31,7 +32,7 @@ locals {
 
 module vpc {
   source                    = "../modules/vpc"
-  tags                      = var.tags
+  tags                      = local.tags
```
Comment (agajjala) [inline Infrastructure/provider/aws/outbound_data_plane/main.tf:35]: thanks for adding these new tags

File: Infrastructure/deployer/deploy.py [hunk 3]
```diff
@@ -73,7 +73,6 @@ def run(args):
     """
     manifest = common.load_yaml(args['manifest'])
     additional_manifest = json.loads(args['manifest_override'])
-    print(additional_manifest)
     manifest.update(additional_manifest)
```
Comment (rnathan) [inline Infrastructure/deployer/deploy.py:76]: test comment

File: Infrastructure/provider/aws/pipeline/buildspecs/dp_apply.yaml [hunk 4]
```diff
@@ -8,7 +8,7 @@ env:
 phases:
   install:
     runtime-versions:
-      python: 3.7
+      python: $PYTHON_VERSION
     commands:
       - curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
```
Comment (rnathan) [inline Infrastructure/provider/aws/pipeline/buildspecs/dp_apply.yaml:13]: Not for this PR: Need to remove the external dependencies. Good to store this internally as a package.

File: Infrastructure/provider/aws/pipeline/buildspecs/tf_apply_outbound.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    GIT_USERNAME: SF-SDN-GIT-TOKEN:GIT_USERNAME
+    GIT_PASSWORD: SF-SDN-GIT-TOKEN:GIT_PASSWORD
+
+phases:
+  install:
+    runtime-versions:
+      python: $PYTHON_VERSION
+    commands:
+      - apt install unzip -y
+      - wget https://releases.hashicorp.com/terraform/"$TF_VERSION"/terraform_"$TF_VERSION"_linux_amd64.zip
```
Comment (rnathan) [inline Infrastructure/provider/aws/pipeline/buildspecs/tf_apply_outbound.yaml:14]: Same as above. Need to remove external dependencies. Good to create a WI to track this.

**Pull request merged**